mail-storage/src/main/java/mail/storage/dto/DateRangeDto.java

```diff
@@ -0,0 +1,17 @@
+package mail.storage.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class DateRangeDto {
+    private Date beginDate;
```

Comment:
it has to be DateTime - otherwise will be a mess in the range: if beginDate is the same as endDate wihtout time it;s easy. Also it's better to store LocalDateTime - becasue time zone differs on the planet

Reply:
Fixed.